src/SIL.XForge.Scripture/Services/ParatextSyncRunner.cs

```diff
@@ -483,13 +483,18 @@ CancellationToken token
                 if (!_paratextService.IsResource(paratextId))
                 {
                     LogMetric("Updating Paratext notes");
+                    if (questionDocs.Count > 0)
```

Comment:
This block gets run, even if no questions are being exported. This means that the note tag is created even if no notes will use it. Is that behavior OK?

Reply:
@pmachapman See the "Paratext note tag integration" document; it has requirements for things like this. One of the things that is specifically called out is to not create a note tag unless it's actually needed to be used. So good catch here (just pointing out the doc so you're aware of it and can see the specs we came up with).

Reply:
Good catch. I've updated the logic. Now it will only create a note tag if there are exportable answers, meaning that there is an answer that is marked exportable, or there are answers and the project setting for exporting all answers is set.